package.json

```diff
@@ -42,10 +43,12 @@
     "@typescript-eslint/eslint-plugin": "5.36.1",
     "@typescript-eslint/parser": "5.36.1",
     "cross-env": "7.0.3",
+    "cspell": "6.8.1",
     "eslint": "8.23.0",
     "eslint-config-prettier": "8.5.0",
     "eslint-plugin-prettier": "4.2.1",
-    "jest": "29.0.1",
+    "husky": "^8.0.0",
+    "jest": "28.0.0",
```

Comment:
Is there any reason that you change `jest` version?

Reply:
yup, ts-jest version is not compatible with jest version.